django/db/models/signals.py

```diff
@@ -15,15 +16,16 @@ class ModelSignal(Signal):
     of the `app_label.ModelName` form.
     """
     def _lazy_method(self, method, apps, receiver, sender, **kwargs):
+        # import Options locally to avoid a circular import
```

Comment:
I'm unsure if this type of comment is needed these days. As far as I know, we only use inner imports to solve circular imports -- any opinion?